packages/filesystem/src/electron-browser/preload.ts

```diff
@@ -21,10 +21,12 @@ import { ipcRenderer, contextBridge } from '@theia/core/electron-shared/electron
 const api: TheiaFilesystemAPI = {
     showOpenDialog: (options: OpenDialogOptions) => ipcRenderer.invoke(CHANNEL_SHOW_OPEN, options),
     showSaveDialog: (options: SaveDialogOptions) => ipcRenderer.invoke(CHANNEL_SHOW_SAVE, options),
+    useNativeDialogs: !('THEIA_ELECTRON_DISABLE_NATIVE_ELEMENTS' in process.env && process.env.THEIA_ELECTRON_DISABLE_NATIVE_ELEMENTS === '1')
```

Comment:
This does not in the TheiaFileSystemAPI object. It is a generalized flag that applies to _all_ electron-based UI and should therefor go into the core package.

Reply:
Sure, you are absolutely right. Moved to core and renamed to `useNativeElements` to simplify moving away from the env variable approach.